org.wso2.carbon.identity.serializer.fst/pom.xml

```diff
@@ -0,0 +1,114 @@
+                        </Export-Package>
+                        <Import-Package>
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            com.fasterxml.jackson.core;version="${com.fasterxml.jackson.package.import.version.range}",
+                            com.fasterxml.jackson.core.io;resolution:=optional;version="${com.fasterxml.jackson.package.import.version.range}",
+                            com.fasterxml.jackson.core.json;resolution:=optional;version="${com.fasterxml.jackson.package.import.version.range}",
+                            com.fasterxml.jackson.core.util;resolution:=optional;version="${com.fasterxml.jackson.package.import.version.range}",
+                            org.apache.commons.logging;version="${commons-logging.osgi.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.*;version="${carbon.identity.framework.package.import.version.range}",
+                            org.wso2.carbon.identity.application.authenticator.*;version="${carbon.identity.application.authenticator.basicauth.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.handler.*;version="${carbon.identity.application.authenticator.basicauth.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.common.*;version="${carbon.identity.framework.package.import.version.range}",
+                            org.wso2.carbon.identity.oidc.session.*;version="${identity.oidc.session.package.import.version.range}
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
+                        <Embed-Dependency>fst|objenesis;scope=compile|runtime;inline=false</Embed-Dependency>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>de.ruedigermoeller</groupId>
+            <artifactId>fst</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.osgi</groupId>
+            <artifactId>org.eclipse.osgi.services</artifactId>
```

Comment:
Why do we need osgi related components as dependencies? Do we use them at compile time?